packages/@aws-cdk/aws-events/test/test.input.ts

```diff
@@ -102,6 +102,49 @@ export = {
       }));
       test.done();
     },
+
+    'can use token from cfnParameter'(test: Test) {
+      // GIVEN
+      const stack = new cdk.Stack();
+     
+      const invokeScheduleInMinutes = new cdk.CfnParameter(stack, 'invokeScheduleInMinutes', {
+        type: 'Number',
+        description: 'The minutes between scheduled invocations.',
+        default: 5,
+      });
+
+      const rule = new Rule(stack, 'Rule', {
+        schedule: Schedule.rate(cdk.Duration.minutes(invokeScheduleInMinutes.valueAsNumber)),
+      });
+      const user = new User(stack, 'User');
+
+      // WHEN
+      rule.addTarget(new SomeTarget(RuleTargetInput.fromObject({ userArn: user.userArn })));
+
+      // THEN
+      expect(stack).to(haveResourceLike('AWS::Events::Rule', {
+        Targets: [
```

Comment:
We should be asserting the `ScheduleExpression` property instead of the `Targets` property, no?

Reply:
Right, my bad.